style.css

```diff
@@ -0,0 +1,140 @@
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  text-align: center;
+}
+
+.menu {
+  display: flex;
+  gap: 30px;
+  justify-content: center;
+  list-style: none;
+  margin: 20px;
+}
+
+.h-menu {
+  display: none;
+}
+
+.body h1 {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.body h2 {
+  color: #36b37e;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+.body p {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+}
```

Comment:
- [ ] You have made a great piece out here, @yegonkimutai but I do think you need to check a couple of things to improve your design even better in order to match requirements..
Kindly consider the designs below:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/218836918-ca280152-64a0-4f3b-bb09-4a3c07cc4b35.png)|![image](https://user-images.githubusercontent.com/99715640/218836996-f4783e58-5915-4d39-ac7a-3af9642a248d.png)|

- [ ] Your headings (`h1` and `h2`) have appropriate font-sizes but I am not sure the `font-weight` is not being conflicted with other properties maybe due to specificity issues. Kindly inspect or use a `id` to style them so that they can be distinct and bold for good user experience and accessibility.
- [ ] Another thing I will suggest you look into is the background images. Kindly make your own very similar to the Figma design. :+1: